client/components/Deposit/DataciteDeposit.tsx

```diff
@@ -85,7 +85,7 @@ export default function DataciteDeposit(props: Props) {
 	const pub = 'pub' in props ? props.pub : undefined;
 
 	useEffect(() => {
-		if (pub && pub.releases.length > 0) {
+		if (pub && (pub.releases?.length ?? 0) > 0) {
```

Comment:
i am somewhat inconsistent with this, sometimes I do this, sometimes I do `pub && pub.releases && ...`. Let me know if i should make that more consistent